acceptance_tests/features/load_ashe_sample_file.feature

```diff
@@ -9,5 +12,5 @@ Feature: Load ASHE sample
   @us081_s001
   Scenario: Load ASHE sample
     Given the 201803 collection exercise for the ASHE survey has been created
-    When the internal user navigates to the collection exercise details page for ASHE 201803
+    When the internal user navigates to the collection exercise details page for ASHE survey
```

Comment:
It feels misleading to leave the wording here talking about a specific ASHE survey and collection exercise while it is now using generic generated data like all the other tests. Could we either make the scenario wording reflect the data it is testing against, or delete this scenario altogether if it's not testing anything relevant anymore?

Reply:
fixed, deleted as you're right, it's not testing ashe specific functionality anymore